packages/fields-document/CHANGELOG.md

```diff
@@ -0,0 +1,277 @@
+# @keystonejs/fields-markdown
```

Comment:
Just a note - this file should probably be deleted as it's not relevant to this fields-document